plugins/available/plenv.plugin.bash

```diff
@@ -0,0 +1,17 @@
+# plugin for plenv
+
+cite about-plugin
+about-plugin 'plenv plugin for Perl'
+
+pathmunge "${HOME}/.plenv/bin"
```

Comment:
Maybe wrap this in an `if` statement as well? Don't want to pollute the path if this doesn't exist...